doc/plugins/org.polarsys.capella.ui.doc/html/04. User Interface/4.13. Activity Explorer.mediawiki

```diff
@@ -0,0 +1,16 @@
+= Activity Explorer =

```

Comment:
It seems to be covered in Capella Overview / Activity Browser.

![image](https://user-images.githubusercontent.com/60220371/128978047-c66cd27a-a913-4a8c-955a-ea88b4d70d69.png)